app.js

```diff
@@ -92,7 +93,15 @@ app.use((err, req, res, next) => {
   logger.error(err, 'burped error');
   res.status(err.status || 500).json({msg: err.message});
 });
+
 const httpServer = app.listen(PORT);
+process.on('uncaughtException', function(err) {
+    if(err.errno === 'EADDRINUSE' && useMultiple) {
+        const httpServer = app.listen(PORT++);
```

Comment:
It seems to me that this error will also be thrown from lib/utils/aws-sns-lifecycle, yet here we are assuming that the bind error can only come from the healthcheck port?